src/KDSoapClient/KDSoapClientInterface.h

```diff
@@ -135,7 +135,7 @@ class KDSOAP_EXPORT KDSoapClientInterface
      *  }
      * \endcode
      */
-    KDSoapPendingCall asyncCall(const QString &method, const KDSoapMessage &message,
+    KDSoapPendingCall asyncCall(const QString &method, KDSoapMessage &message,
```

Comment:
This interface change is unwanted. It could break existing code,

Reply:
Yes i have faced with it when started to work on the unit test. Fixed.